scripts/build_zlib.sh

```diff
@@ -6,17 +6,21 @@ function usage() {
     echo "Usage: `basename $0` [-t <Release|Debug>]"
     echo "Builds zlib"
     echo "-t <Release/Debug> : Release or Debug builds"
+    echo "-v: return version"
```

Comment:
add `-v` option to return the version of the component. Each build_NNNN.sh should implement this.